SONAR-netCDF4/docs/crr341.adoc

```diff
@@ -426,22 +428,19 @@ There are five right-handed Cartesian coordinate systems (<<coordinateSystemList
 |Transducer coordinate system|Centre of transducer face|Pointing forward along transducer plane|Pointing starboard along transducer plane|Pointing down orthogonal to transducer plane
 |Sonar beam coordinate system|Centre of transducer face|Arbitrary|Orthogonal to the _x_-axis|Pointing along the beam axis
 |===
 
-The surface coordinate system (SCS) is obtained from the fixed coordinate system (FCS) through a translation of the origin (e.g., latitude and longitude assuming the FCS has latitude of 0 degrees and longitude of 0 degrees) and a rotation about the _z_-axis (typically the platform heading).
-
-The platform coordinate system (PCS) is obtained from the SCS through a rotation that corresponds to the pitch, roll of the platform. No translation is necessary.
+The platform coordinate system (PCS) is obtained from the surface coordinate system (SCS) through a rotation that corresponds to the pitch and roll of the platform. No translation is necessary. Platform heave is applied in this coordinate system.
 
-The transducer coordinate system (TCS) is obtained from the PCS through a translation of the PCS origin and a rotation to align with the transducer face.
+The transducer coordinate system (TCS) is obtained from the PCS through a translation of the PCS origin and a rotation to align with the transducer face. Transducer depth is applied in this coordinate system.
 
-The sonar beam coordinate system (BCS) is obtained from the TCS through a rotation that specifies any difference in transducer pointing direction and beam pointing direction. In many cases there will be no rotation necessary. Note that split-aperture coordinates (minor and major angles) are defined in terms of the BCS and a rotation may be necessary.
+The sonar beam coordinate system (BCS) is obtained from the TCS through a rotation that specifies any difference in transducer pointing direction and beam pointing direction. In many cases there will be no rotation necessary. Note that split-aperture coordinates (minor and major angles) are defined by the BCS and a rotation may be necessary to give the desired minor and major angle directions.
 
-If a sonar actively alters beam pointing directions to compensate for motion of the platform, the rotations to get from the TCS to the BCS will be identical to those given to get from SCS to PCS, but with opposite sign (_is this correct? should there be a flag to avoid having to store these pitch, roll, heading values twice?_).
+If a sonar actively alters beam pointing directions to compensate for motion of the platform, the rotations to get from the TCS to the BCS will be identical to those given to get from SCS to PCS, but with opposite sign. The _beam_stabilisation_ variable indicates whether the beams were compensated or uncompensated and serves as information only (the rotation matrices are still used in the same way).
 
```

Comment:
comment: this is not very clear to us and we would recommend to define beam pointing angle in SCS coordinate system if the sonar actively alters beam pointing directions to compensate for motion of the platform and in TCS if the sonar does not actively alters beam pointing directions to compensate for motion of the platform. The _beam_stabilisation_ variable indicates whether the beams were compensated or uncompensated and changes the formula for positionning echoes with rotation matrices.
We can then give the formula for positionning echoes in geographical coordinate system using the Sonar NetCDF4 convention attributes for the two cases (eg EK80 uncompensated for motion sensor and eg ME70 compensated for motion sensor)

Reply:
I did struggle with doing this in a clear way and your suggestion resolves that :)